src/modules/auth/actions/update-assets.js

```diff
@@ -27,8 +27,9 @@ export function updateAssets(callback = logError) {
         if (allAssetsLoaded(balances)) callback(null, balances)
       })
     })
-    augur.rpc.eth.getBalance([loginAccount.address, 'latest'], (attoEthBalance) => {
-      if (!attoEthBalance || attoEthBalance.error) return callback(attoEthBalance)
+    augur.rpc.eth.getBalance([loginAccount.address, 'latest'], (err, attoEthBalance) => {
+      if (err) return callback(err)
+      if (attoEthBalance == null) return callback(attoEthBalance)
```

Comment:
Missing test coverage of the 'attoEthBalance' callback.

Reply:
I don't think that eth.getBalance can return null or undefined. Even for a brand new account it should just return 0